node-commons/src/main/java/org/apache/iotdb/commons/partition/DataPartition.java

```diff
@@ -238,11 +248,73 @@ public void createDataPartition(
         .put(timePartitionSlot, Collections.singletonList(regionReplicaSet));
   }
 
-  public void serialize(ByteBuffer buffer) {
-    // TODO: Serialize DataPartition
+  public void serialize(ByteBuffer buffer) throws IOException, TException {
```

Comment:
Don't forget to add tests for serialize~

Reply:
fixed.